Utilities/DataLoading/EdFi.LoadTools.Test/SmokeTests/APIGetTests.cs

```diff
@@ -156,12 +183,11 @@ public void GetAll_should_store_results_in_dictionary()
         [Test]
         public void GetSkipLimitTest_should_retrieve_second_object()
         {
-            var dictionary = new Dictionary<string, JArray>
-                             {
-                                 [ResourceName] = _data
-                             };
+            var dictionary = new Dictionary<string, JArray> {[ResourceName] = _data};
+            var client = Host.GetTestClient();
+            client.BaseAddress = new System.Uri(Address);
 
-            var subject = new GetAllSkipLimitTest(_resource, dictionary, _configuration, tokenHandler);
+            var subject = new GetAllSkipLimitTest(_resource, dictionary, _configuration, tokenHandler, client);
             var result = subject.PerformTest().Result;
```

Comment:
this should be await subject.PerformTest()

Reply:
Updated